tox.ini

```diff
@@ -47,60 +49,66 @@ max-line-length = 120
 ignore = E121 E126 E123 E128 E133 E226 E241 E242 E704 W291 W293 W391 W503 W504 W505
 
 [testenv:docs]
-base_python = python3.8
+base_python = python3.10
 skip_install = true
 deps =
     -r requirements-dev.txt
 commands =
     python scripts/make_docs.py
 
 [testenv:coverage]
-base_python = python3.8
+base_python = python3.10
 skip_install = true
 deps =
     -r requirements-dev.txt
 commands =
     python scripts/make_coverage.py
 
 [testenv:benchmarks]
-base_python = python3.8
+base_python = python3.10
 deps =
     -r requirements-dev.txt
 commands =
     python benchmarks/benchmark_all_examples.py
 
 [testenv:profiling]
-base_python = python3.8
+base_python = python3.10
 deps =
     -r requirements-dev.txt
 commands =
     python benchmarks/profile_all_examples.py
 
 [testenv:examples]
-base_python = python3.8
+base_python = python3.10
 deps =
     -r requirements-dev.txt
 commands =
     python benchmarks/run_all_examples.py
 
 [testenv:notebooks]
-base_python = python3.8
+base_python = python3.10
 allowlist_externals =
     bash
     pwd
 deps =
     -r requirements-dev.txt
 commands =
-    bash -c "pwd"
-    bash -c "echo $PYTHONPATH"
-    python -m jupyter nbextension install --py --sys-prefix widgetsnbextension
-    python -m jupyter nbextension enable --py --sys-prefix widgetsnbextension
-    python -m jupyter nbextension install --py --sys-prefix jpy_canvas
-    python -m jupyter nbextension enable --py --sys-prefix jpy_canvas
-    python notebooks/run_all_notebooks.py
+    # Replaced `run_all_notebooks.py` with nbmake, and this explicit list of notebooks:
+    python -m pytest --nbmake  \
+        notebooks/simple-graph-plot-2022.ipynb \
+        notebooks/Agent-Close-Following.ipynb \
+        notebooks/simple_example_manual_control.ipynb \
+        notebooks/simple_rendering_demo.ipynb \
+        notebooks/flatland_animate.ipynb \
+        notebooks/render_episode.ipynb \
+        notebooks/scene_editor.ipynb \
+        notebooks/test_saved_envs.ipynb 
+    # need to enable redis (in a container) in github workflow for this to work
+    #    notebooks/test_service.ipynb
 
 [testenv:start_jupyter]
-base_python = python3.8
+# TODO: What is this for?  I think we could delete it. Jeremy.
```

Comment:
Yeah, I guess I just ported this from the old setup. If we don't need it, let's delete it. @manuschn maybe you know more?